### PR TITLE
Feat/dgds disclaimer

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: static/docs/cookie-agreements.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Cookies
+&nbsp;
+
+Cookies are small files containing an identity code, which are sent to your device and stored on your browser or hard drive. These cookies allow us to distinguish you from another user and provide you with a unique browsing experience. The cookies will not cause harm to your device.
+&nbsp;
+
+Most browsers allow you to prevent (from accepting) certain cookies, to require your consent prior to storing a cookie, or block cookies altogether through your browser’s privacy preferences menu. Please find out more about controlling cookies on www.aboutcookies.org.
+&nbsp;
+
+Deltares uses two types of cookies for this website: functional cookies (for making the website work properly), and analytical cookies in anonymous mode.
```
Comment (SiggyF) [inline static/docs/cookie-agreements.md]: gebruiken  we die? Of  later?

File: components/disclaimer.vue [hunk 2]
```diff
@@ -0,0 +1,178 @@
+<template>
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/disclaimer.vue]: I agree completely; 
I want to take this on in another ticket to look at naming conventions (file names, class names etc). 
Changed it now only for the disclaimer component

File: components/disclaimer.vue [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  },
+  validations: {
+    cookie: {
+      required
+    },
+    agree: {
+      required
+    }
+  },
+  mounted () {
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/disclaimer.vue]: replaced by require and a markdown loader

File: components/disclaimer.vue [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+              I agree with the Conditions of Use
+            </UiCheckbox>
+            <UiCheckbox
+              :checked="cookie"
+              @input="setAgreementTarget($event, 'cookie')"
+            >
+              I consent with the use of cookies
+            </UiCheckbox>
+          </div>
+          <div class="modal-content-actions__agree-button">
```
[the remark this answers is not in the document]
Comment (SiggyF) [reply components/disclaimer.vue]: The current approach is probably based  on [BEM](http://getbem.com/naming/). Discuss later.

File: components/disclaimer.vue [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+  align-items: center;
+  flex-grow: 1;
+}
+
+.modal-content-text__markdown {
+  padding-bottom: var(--spacing-small);
+}
+
+.modal-content__text {
+  font-size: unset;
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/disclaimer.vue]: Oh was an earlier attempt of getting the markdown in there correctly. I removed it now

File: components/disclaimer.vue [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+.modal-content__actions {
+  flex-grow: 1;
+  display: flex;
+  padding-top: var(--spacing-small);
+}
+
+.modal-content-actions__checkboxes {
+  flex-grow: 9;
+  display: flex;
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/disclaimer.vue]: yes

File: components/disclaimer.vue [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+  flex-direction: column;
+}
+
+.modal-content-actions__agree-button {
+  margin-top: auto;
```
[the remark this answers is not in the document]
Comment (CindyvdVries) [reply components/disclaimer.vue]: no, this pushes the button down to the bottom

File: static/docs/cookie-agreements.md (hunk 1 above)
Comment (CindyvdVries) [reply static/docs/cookie-agreements.md]: I think later. 
The text in the cookie message needs to be addepted by Daniel. This is incomplete at the moment

**Pull request merged**